set_default.cpp

```diff
@@ -238,6 +238,9 @@ void set_default_input_values(void) {
   g_sequential_source_gamma_id_list[15] = _default_seq_source_gamma_id;
 
   g_sequential_source_gamma_id_number = _default_seq_source_gamma_id_number;
+  g_spintrace = _default_spintrace;
+  g_filtered_qsq = _default_filtered_qsq ;
+  g_currentgammas = cvc::UKQCD ;
```

Comment:
can you rename this to g_current_gamma_basis?

Reply:
or, even better, g_matrix_gamma_basis?

Reply:
or even better, one could name everything to do with the spin projection with that prefix:

```
<SPINPROJECTIONGAMMABASIS>
```

etc.. what do you think?

Reply:
you mean for example g_spinprojection_gamma_basis?

Reply:
Yes, also for the name of the parameter etc., just so it's completely clear that this only affects the spin projection.